### ISSUE TITLE
Argument count for macros is not correctly determined.

### ISSUE THREAD

**Issue by billw2012** [SQFvm/vm#40]
Commas within strings passed to macros will cause the count to be wrong and generate "Arg Count Missmatch".
Repro:
```sqf
#define MACRO_TEST(a, b, c)
MACRO_TEST("this is a comma, test", 1, 2);
```

**Comment by billw2012** [comment]
It doesn't handle escaped quotes:
```
#define MACRO_TEST(a, b, c)
MACRO_TEST("this is a com""adwda""ma, test", 1, 2);
```
`[ERR][L4|C50|c:\Users\billw\Documents\Arma 3\mpmissions\Project_0\tools\sqf-vm-test.sqf]        Arg Count Missmatch.`

**Comment by X39** [comment]
Fixed too in f701fd2c3cd0a8f8fbb1c8dbd148442b24ea6b16
